core/scripts/cli/vote.js

```diff
@@ -0,0 +1,38 @@
+const inquirer = require("inquirer");
+
+const vote = async () => {
+  const prompts = [
+    {
+      type: "list",
+      name: "voteTopMenu",
+      message: "Voting actions",
+      choices: ["list pending price requests and vote", "commit specific vote", "reveal votes", "get round ID", "back"]
+    }
+  ];
+
+  return await inquirer.prompt(prompts);
+};
+
+module.exports = async function(web3, artifacts) {
+  const Voting = artifacts.require("Voting");
+  const voting = await Voting.deployed();
+  try {
+    const inputs = await vote();
+    switch (inputs["voteTopMenu"]) {
+      case "list pending price requests and vote":
+        const pendingRequests = await voting.getPendingRequests();
```

Comment:
Is there ever a delay here to make a `.call()` action to the blockchain? What are your thoughts on alerting the user that we are currently "fetching" data

Reply:
ye, that's a good point. I was thinking we could use a spinner of some kind while the promise resolves.

Reply:
+1 good idea